packages/cli/src/databases/entities/Project.ts

```diff
@@ -23,3 +30,66 @@ export class Project extends WithTimestampsAndStringId {
 	@OneToMany('SharedWorkflow', 'project')
 	sharedWorkflows: SharedWorkflow[];
 }
+
+@EventSubscriber()
+export class UserSubscriber implements EntitySubscriberInterface<User> {
+	listenTo() {
+		return User;
+	}
+
+	async afterUpdate(event: UpdateEvent<User>): Promise<void> {
+		if (event.entity) {
+			const newUserData = event.entity;
+
+			if (event.databaseEntity) {
+				const fields = event.updatedColumns.map((c) => c.propertyName);
+
+				if (
+					fields.includes('firstName') ||
+					fields.includes('lastName') ||
+					fields.includes('email')
+				) {
+					const oldUser = event.databaseEntity;
+					const name = `${newUserData.firstName} ${newUserData.lastName} <${newUserData.email}>`;
+
+					const project = await Container.get(ProjectRepository).getPersonalProjectForUser(
+						oldUser.id,
+					);
+
+					if (!project) {
+						// Since this is benign we're not throwing the exception. We don't
+						// know if we're running inside a transaction and thus there is a risk
+						// that this could cause further data inconsistencies.
+						const message = "Could not update the personal project's name";
+						Container.get(Logger).warn(message, event.entity);
+						const exception = new ApplicationError(message);
+						// TODO: not sure this is the right way to report something custom to sentry
+						captureException(exception, event.entity);
```

Comment:
@krynble Is this the correct way to send an error to sentry?

Reply:
You should use the `ErrorReporterProxy` instead - in case sentry is set up, it sends to sentry, otherwise it outputs to terminal. This way, it could fail because DSN might not be set.

Reply:
done